tests/src/main/java/com/hazelcast/simulator/tests/map/IntByteMapTest.java

```diff
@@ -77,11 +79,17 @@ public void tearDown() throws Exception {
 
     @Warmup(global = false)
     public void warmup() throws InterruptedException {
-        MapStreamer<Integer, Object> streamer = MapStreamerFactory.getInstance(map);
         Random random = new Random();
+        values = new byte[valueCount][];
+        for (int k = 0; k < values.length; k++) {
+            int delta = maxSize - minSize;
```

Comment:
for the record - I'd pull-out a method out of this loop: `generateValue(minLength, maxLength)` 
I appreciate it's matter of taste, it's not a must.